website/source/docs/agent/options.html.markdown

```diff
@@ -499,12 +499,26 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
   using TCP to get the full set of records.
 
   * <a name="only_passing"></a><a href="#only_passing">`only_passing`</a>
-    When set to true, the default, the querying agent will only receive node
-    or service addresses for healthy services.  A healthy service is defined
-    as a service with one or more healthchecks and all defined healthchecks
-    for the service are in a passing or warning state (i.e. not
-    critical). Set to false to have the querying agent include all node and
-    service addresses regardless of the health of the service.
+  When set to true, the default, the querying agent will only receive node
+  or service addresses for healthy services.  A healthy service is defined
+  as a service with one or more healthchecks and all defined healthchecks
+  for the service are in a passing or warning state (i.e. not
+  critical). Set to false to have the querying agent include all node and
+  service addresses regardless of the health of the service.
+
+  * <a name="udp_answer_limit"></a><a
+  href="#udp_answer_limit">`udp_answer_limit`</a> - Limit the number of
+  resource records contained in the answer section of a UDP-based DNS
+  response.  The default number of resource records returned is `3`.  In
+  environments where
+  [RFC 3484 Section 6](https://tools.ietf.org/html/rfc3484#section-6) Rule 9
+  is implemented and enforced (i.e. DNS answers are always sorted and
+  therefore never random), clients may need to set this value to `1` to
+  preserve randomized DNS round-robin (note:
```

Comment:
I'd say "load balancing" instead of "round-robin" since we don't technically do that.